aptos-move/move-examples/tests/move_unit_tests.rs

```diff
@@ -92,3 +92,15 @@ fn test_shared_account() {
     )]);
     run_tests_for_pkg("shared_account", named_address);
 }
+
+#[test]
+fn test_mint_nft() {
+    let named_address = BTreeMap::from([(
+        String::from("mint_nft"),
+        AccountAddress::from_hex_literal(
+            "0x0b6beee9bc1ad3177403a04efeefb1901c12b7b575ac5124c0205efc0dd2e32a",
```

Comment:
could we use the account_address::create_resource_address?

Reply:
sounds good!